Cargo.toml

```diff
@@ -40,6 +40,17 @@ homepage = "https://github.com/kanidm/kanidm/"
 repository = "https://github.com/kanidm/kanidm/"
 
 [workspace.dependencies]
+kanidmd_core = { path = "./server/core" }
+kanidmd_lib = { path = "./server/lib" }
+kanidmd_lib_macros = { path = "./server/lib-macros" }
+kanidm_build_profiles = { path = "./libs/profiles", version = "1.1.0-rc.14-dev" }
+kanidm_lib_crypto = { path = "./libs/crypto" }
+kanidm_lib_file_permissions = { path = "./libs/file_permissions" }
+kanidm_client = { path = "./libs/client", version = "1.1.0-rc.14-dev" }
```

Comment:
you shouldn't need the "version" bit if you've got the path?

Reply:
Need it for publishing though.